app/assets/stylesheets/mobile/lightbox.scss

```diff
@@ -0,0 +1,19 @@
+.lightbox:hover .meta {
```

Comment:
There's no `:hover` on mobile.

Reply:
Oddly enough it does seem to have an effect. You wouldn't think it would! If you remove the `:hover` pseudo-class the expand icon will show at all times. If you have `:hover` in place, the expand icon will only show when a finger swipe occurs over a lightboxed image. This is at least how it works on Safari on the iPhone.

Reply:
I know it does something, just not what we expect ;) Anyway, on mobile we always want to show the expand arrows.

Reply:
Done. The new arrows will be visible on mobile at all times with an opacity of 0.8.

I had to add `.lightbox:hover .meta` because otherwise if the user would hold and slide their finger on the image it would the opacity would become 1. I wanted a consistent behavior.